extensions/stackdriver/log/logger.cc

```diff
@@ -31,6 +32,13 @@ namespace Extensions {
 namespace Stackdriver {
 namespace Log {
 namespace {
+// Matches Rbac Access denied string.
+// It is of the format:
+// "rbac_access_denied_matched_policy[ns[NAMESPACE]-policy[POLICY]-rule[POLICY_INDEX]]"
+const RE2 rbac_denied_match(
+    "rbac_access_denied_matched_policy\\[ns\\[(.*)\\]-policy\\[(.*)\\]-rule\\[("
+    ".*)\\]\\]");
+constexpr char kRbacAccessDenied[] = "RbacAccessDenied";
```

Comment:
will this be exposed directly to user on UI? If so we should call it something like "AuthzDenied", otherwise it's fine either way if it is used internally only.

Reply:
Yes, fixed